packages/synapse-react-client/src/components/ShareThisPage/ShareThisPage.tsx

```diff
@@ -0,0 +1,117 @@
+import {
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  IconButton,
+  Typography,
+  Input,
+  Box,
+} from '@mui/material'
+import { useState } from 'react'
+import { ReactComponent as ShareIcon } from '@/assets/icons/sharethispage.svg'
+import CloseIcon from '@mui/icons-material/Close'
+import CopyToClipboardIcon from '../CopyToClipboardIcon'
+import {
+  LinkedinShareButton,
+  LinkedinIcon,
+  TwitterShareButton,
+  TwitterIcon,
```

Comment:
TwitterIcon appears dated, update to X icon?  https://about.x.com/en/who-we-are/brand-toolkit